### PR TITLE
Downcase invalid ACIdentifier rkt.kubernetes.io/terminationMessagePath

### PR DESCRIPTION
**Pull request by shanegibbs**
Conform to the ACI spec at [acidentifier.go](https://github.com/appc/spec/blob/3972f6a1b657f38e30100d2a7ef377cc8690c8db/schema/types/acidentifier.go#L25-L29).

### CODE REVIEW

File: pkg/kubelet/rkt/rkt.go [hunk 1]
```diff
@@ -81,7 +81,7 @@ const (
 	k8sRktCreationTimeAnno           = "rkt.kubernetes.io/created"
 	k8sRktContainerHashAnno          = "rkt.kubernetes.io/containerhash"
 	k8sRktRestartCountAnno           = "rkt.kubernetes.io/restartcount"
-	k8sRktTerminationMessagePathAnno = "rkt.kubernetes.io/terminationMessagePath"
+	k8sRktTerminationMessagePathAnno = "rkt.kubernetes.io/terminationmessagepath"
```
Comment (dchen1107) [inline pkg/kubelet/rkt/rkt.go]: how about termination-message-path?

Comment (shanegibbs) [inline pkg/kubelet/rkt/rkt.go]: I agree that `terminationmessagepath` is getting a bit long. But it seems to be the standard here.

I would be reluctant to change it to `termination-message-path` without changing the rest of them, `restartcount` to `restart-count` for example.

Comment (shanegibbs) [inline pkg/kubelet/rkt/rkt.go]: I will defer to your expertise if you think this is best however.

**Comment by shanegibbs** [comment]
@googlebot We have signed the CLA as a corporation. Company name is "Sittercity Incorporated".

File: pkg/kubelet/rkt/rkt.go (hunk 1 above)
Comment (dchen1107) [inline pkg/kubelet/rkt/rkt.go]: yes, let's change rest of annotation too. Thanks!

**Comment by shanegibbs** [comment]
@dchen1107 do you know if this failed test was due to the code change? or the build infrastructure?

Are we able to run it again?

**Comment by mikedanese** [comment]
is this a breaking change?
cc @yifan-gu

**Comment by shanegibbs** [comment]
Closing due to PR https://github.com/kubernetes/kubernetes/pull/20816

**Comment by yifan-gu** [comment]
@shanegibbs Thank you! I updated https://github.com/kubernetes/kubernetes/pull/20816 as your PR suggested here.

**Pull request closed without merging**